pkg/cmd/roachtest/restore.go

```diff
@@ -360,18 +360,20 @@ func registerRestore(r *testRegistry) {
 				// 	})
 				// })
 
+				tick := initBulkJobPerfArtifacts(ctx, testName, item.timeout)
 				m.Go(func(ctx context.Context) error {
 					defer dul.Done()
 					defer hc.Done()
 					t.Status(`running restore`)
 					c.Run(ctx, c.Node(1), `./cockroach sql --insecure -e "CREATE DATABASE restore2tb"`)
 					// TODO(dan): It'd be nice if we could keep track over time of how
 					// long this next line took.
+					tick()
```

Comment:
A comment here and at the other callsites explaining that we tick before and after the run so that roachperf can look at the total elapsed time between the ticks would probably be beneficial.

Reply:
done.